ds_judgements_public_ui/sass/includes/_search.scss

```diff
@@ -112,10 +118,10 @@
   }
 
   &__header-text {
-    margin-left: 3rem;
+    line-height: 25px;
 
     @media (min-width: $grid-breakpoint-small) {
-      margin-left: 0.75rem;
+      margin-left: 2.75rem;
```

Comment:
If we've aligned the header text left, we shouldn't need this margin left - I'll pull the branch back down and take a look